server/game/cards/14.3-IPoT/Shori.js

```diff
@@ -0,0 +1,21 @@
+const DrawCard = require('../../drawcard.js');
+const AbilityDsl = require('../../abilitydsl');
+
+class Shori extends DrawCard {
+    setupCardAbilities() {
+        this.attachmentConditions({
+            myControl: true,
```

Comment:
my control clause doesn't seem to be in the description?

Reply:
Whoops, made assumptions based on the other two